accounts/keystore/keystore.go

```diff
@@ -138,7 +138,7 @@ func (ks *KeyStore) refreshWallets() {
 
 	// Transform the current list of wallets into the new one
 	wallets := make([]accounts.Wallet, 0, len(accs))
-	events := []accounts.WalletEvent{}
+	var events []accounts.WalletEvent
```

Comment:
Please merge these two declarations into 

```go
var (
    wallets = make([]accounts.Wallet, 0, len(accs))
    events  []accounts.WalletEvent
)
```

It looks weird to have both types one beside the other.

Reply:
Done.